### PR TITLE
added packages/dokeysto/dokeysto.3.0.1

### PR DESCRIPTION
**Pull request by UnixJunkie**
minor update

### CODE REVIEW

**Comment by UnixJunkie** [comment]
In one CI test, the camlp4 installation failed.
I guess that's not the fault of my package.

File: packages/dokeysto_camltc/dokeysto_camltc.3.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "dokeysto_camltc"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
```
Comment (mseri) [inline packages/dokeysto_camltc/dokeysto_camltc.3.0.1/opam]: Is this still using `jbuilder` or this invocation needs to be updated?

Comment (UnixJunkie) [reply packages/dokeysto_camltc/dokeysto_camltc.3.0.1/opam]: Nice catch. I updated it.

**Comment by mseri** [comment]
Thanks

**Pull request merged**